src/planning.py

```diff
@@ -122,19 +122,28 @@ def task_plan(init_atoms: Set[GroundAtom],
     ground_nsrts = utils.filter_static_operators(ground_nsrts, init_atoms)
     nonempty_ground_nsrts = [nsrt for nsrt in ground_nsrts
                              if nsrt.add_effects | nsrt.delete_effects]
-    if not utils.is_dr_reachable(nonempty_ground_nsrts, init_atoms, goal):
+    all_reachable_atoms = utils.get_reachable_atoms(nonempty_ground_nsrts,
+                                                    init_atoms)
```

Comment:
Let's rename `atoms`-> `init_atoms` in sesame_plan() above, partially for consistency with task_plan() here, and also because it's a more clear name.